Dockerfile

```diff
@@ -6,6 +6,6 @@ ENV TERRAFORM_VERSION=1.4.4
 RUN wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
 RUN unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip
 RUN mv terraform /usr/bin/terraform
-
+COPY runtime/target/lib/ lib/
```

Comment:
I think its better we download the file here also directly from internet and delete the jar from our codebase.

Reply:
Ok. I deleted the jar from codebase and download it into the workdir by URL.